src/cli/util.js

```diff
@@ -443,7 +443,13 @@ function formatFiles(context) {
   }
 
   eachFilename(context, context.filePatterns, filename => {
-    const fileIgnored = ignorer.filter([filename]).length === 0;
+    let fileIgnored = false;
+    try {
+      fileIgnored = ignorer.ignores(path.relative(process.cwd(), filename));
+    } catch (_) {
+      // Invalid path
```

Comment:
Would it be better to do this update in `next` instead and show this error to the user? I’m not sure what an “invalid path” is, but it sounds useful to get to know that some of my patterns are wrong?

Reply:
I don't know. I'm not changing current behave

Reply:
@evilebottnawi Do you have experience from ESLint or stylelint?